<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWalletWizardPagePasswordGpg</class>
 <widget class="QWidget" name="KWalletWizardPagePasswordGpg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>385</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Various applications may attempt to use the KDE wallet to store passwords or other information such as web form data and cookies. If you would like these applications to use the wallet, you must enable it now and choose method for its encryption.&lt;/p&gt;&lt;p&gt;GPG method is more secure, but you must have configured at least one encrypting key on your system.&lt;/p&gt;&lt;p&gt;If you choose the classic format, be warned that the password you choose &lt;span style=&quot; font-style:italic;&quot;&gt;cannot&lt;/span&gt; be recovered if it is lost, and will allow anyone who knows it to obtain all the information contained in the wallet.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="_useWallet">
     <property name="text">
      <string>Yes, I wish to use the KDE wallet to store my personal information.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>What kind of encryption do you wish?</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="_radioGpg">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Use GPG encryption, for better protection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_radioBlowfish">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Classic, blowfish encrypted file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Enter a new password:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
              </property>
              <property name="buddy">
               <cstring>_pass1</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel2_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Verify password:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
              </property>
              <property name="buddy">
               <cstring>_pass2</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="KLineEdit" name="_pass1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KLineEdit" name="_pass2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>98</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="_matchLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignAbsolute|Qt::AlignBottom|Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignHorizontal_Mask|Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing|Qt::AlignVCenter|Qt::AlignVertical_Mask</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_radioBlowfish</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioBlowfish</sender>
   <signal>toggled(bool)</signal>
   <receiver>_pass1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioBlowfish</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel2_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioBlowfish</sender>
   <signal>toggled(bool)</signal>
   <receiver>_pass2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioBlowfish</sender>
   <signal>clicked()</signal>
   <receiver>_pass1</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>_groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>_radioGpg</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_useWallet</sender>
   <signal>toggled(bool)</signal>
   <receiver>_radioBlowfish</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
